tutorials/spectral/plot_SpectralVariance.py

```diff
@@ -77,11 +80,11 @@
 ###################################################################################################
 
 # Calculate the spectral histogram
-freqs, bins, spect_hist = spectral.compute_spectral_hist(sig, fs, nbins=50, f_range=(0, 80),
+freqs, bins, spect_hist = compute_spectral_hist(sig, fs, nbins=50, f_range=(0, 80),
```

Comment:
lint comment: can you fix the spacing on second line (also on next function call).

Reply:
Indentation fixed.